samples/web/content/apprtc/chrome-app-build/fetch-fonts.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+
+import os
+import argparse
+import sys
+import re
+import urllib2
+from urlparse import urlparse
+from os.path import basename
+
+parser = argparse.ArgumentParser(description='Parse css file and retrieve all referenced font files')
```

Comment:
Can we split the css font related code into a separate CL? It doesn't seem a priority to me for now.